available_images.md

```diff
@@ -231,6 +231,7 @@ HuggingFace Text Generation Inference Containers
 
 | Framework                          | Job Type   | CPU/GPU | Python Version Options | Example URL                                                                                                              |
 |------------------------------------|------------|---------|------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| PyTorch 2.0.1 with HuggingFace TGI | inference  | GPU     | 3.9 (py39)             | 763104351884.dkr.ecr.us-east-1.amazonaws.com/huggingface-pytorch-tgi-inference:2.0.1-tgi1.1.0-gpu-py39-cu118-ubuntu20.04 |
 | PyTorch 2.0.1 with HuggingFace TGI | inference  | GPU     | 3.9 (py39)             | 763104351884.dkr.ecr.us-east-1.amazonaws.com/huggingface-pytorch-tgi-inference:2.0.1-tgi1.0.3-gpu-py39-cu118-ubuntu20.04 |
 | PyTorch 2.0.1 with HuggingFace TGI | inference  | GPU     | 3.9 (py39)             | 763104351884.dkr.ecr.us-east-1.amazonaws.com/huggingface-pytorch-tgi-inference:2.0.1-tgi0.9.3-gpu-py39-cu118-ubuntu20.04 |
```

Comment:
can you remove older versions like tgi1.0.3, tgi0.9.3 if they are not supported?

Reply:
TGI 0.9.3 is released on 2023/03.  They are all still supported